### PR TITLE
Init buildbot-nix with x86_64-linux worker

### PR DESCRIPTION
**Pull request by steveej**
- [x] depends on #149

### CODE REVIEW

**Comment by r-vdp** [comment]
The flake lock changes are a bit suspicious, I thought I already bumped those inputs.
Is this due to a rebase that undid some of those bumps?

**Comment by steveej** [comment]
> The flake lock changes are a bit suspicious, I thought I already bumped those inputs. Is this due to a rebase that undid some of those bumps?

this branch isn't rebased on develop (yet). i only opened the PR to test buildbot-nix webhooks and builds. 

i was going to finish #143 and then rebase this PR. now that you closed that without merging, how do you want to proceed here?

**Comment by r-vdp** [comment]
I didn't close it, I thought you did.

Probably github did that instead of changing the base branch?

I think we can just reopen it.

File: modules/flake-parts/nixosConfigurations.buildbot-nix-0/configuration.nix [hunk 1]
```diff
@@ -111,4 +127,72 @@
...
+  sops.secrets.buildbot-github-webhook-secret = { };
+  sops.secrets.buildbot-nix-workers = { };
+  # sops.secrets.cachix-auth-token = {};
+
+  services.buildbot-nix.master = {
+    enable = true;
+    admins = [
+      "steveej"
+      "evangineer"
+      "v-rdp"
```
Comment (r-vdp) [inline modules/flake-parts/nixosConfigurations.buildbot-nix-0/configuration.nix]: 🧐

**Pull request merged**